containers/server-saline-image/run_saline.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+if [[ ! -z "$TZ" ]]; then
+    timedatectl set-timezone $TZ
+fi
+
+cp -r /etc/saline.defaults/saline* /etc/salt/
+
+mkdir -p /run/salt/master
+chown -R salt:salt /run/salt
+
+mkdir -p /etc/salt/pki/saline
+
+if [[ "$NOSSL" == "YES" ]]; then
+    cat > /etc/salt/saline.d/restapi.conf <<EOL
+restapi:
+  host: 0.0.0.0
+  disable_ssl: true
+EOL
+else
+    cp /etc/pki/tls/certs/spacewalk.crt /etc/salt/pki/saline/saline.crt
+    cp /etc/pki/tls/private/spacewalk.key /etc/salt/pki/saline/saline.key
+    chown -R salt:salt /etc/salt/pki/saline
+    chmod 0750 /etc/salt/pki/saline
+    cat > /etc/salt/saline.d/restapi.conf <<EOL
+restapi:
+  host: 0.0.0.0
+  ssl_crt: /etc/salt/pki/saline/saline.crt
+  ssl_key: /etc/salt/pki/saline/saline.key
+EOL
+fi
+
+chown -R salt:salt /etc/salt/saline*
+
+exec /usr/bin/salined -l info
```

Comment:
I think it's running as root, if you want to run it as `salt`, use gosu -> https://github.com/tianon/gosu

Reply:
It's switching to salt user internally, the same way as `salt-master` does, because there is some steps performed by core salt functions which are handling the internal process management, don't remember the all details, but in case of changing it it will require significant changes internally and maybe not just in saline code, but in salt as well or it pushes me to rewrite some code of salt and put it to saline directly, but I would like to keep as much as possible inherited from salt.

Reply:
ok